docs/howto/manage-alerts/index.md

```diff
@@ -0,0 +1,40 @@
+# Managing alerts configured with Jsonnet
+
+In addition to [](#uptime-checks), we also have a set of alerts that are configured in support deployments using [](#topic/jsonnet).
+
+## Configuration
+
+We use the [Prometheus alert manager](https://prometheus.io/docs/alerting/latest/overview/) to set up alerts that are defined in the `helm-charts/support/values.jsonnet` file.
+
+At the time of writing, we have the following classes of alerts:
+
+1. when a persistent volume claim (PVC) is approaching full capacity
+2. when a pod has restarted in the last hour.
+
+When an alert threshold is crossed, an automatic notification is sent to PagerDuty and the `#pagerduty-notifications` channel on the 2i2c Slack.
+
+## When a PVC is approaching full capacity
+
+We monitor the capacity of the following volumes:
+
+- home directories
+- hub database
+- prometheus database
+
+The alert is triggered when the volume is more than 90% full.
+
+To resolve the alert, follow the guidance below
+
+- [](../../howto/filesystem-management/increase-disk-size.md)
+- **TODO: add instructions for hub database**
```

Comment:
Help needed: I am unsure if we have docs for this written down somewhere, and what to do if not.

Reply:
@jnywong I don't think we have specific instructions, I'd recommend creating an issue here, linking to this and moving forward. Also mostly if the hub db disk runs out of disk space, it's almost always because of log files - so mostly we'll have to trim the log file